pkg/supply/services/services.go

```diff
@@ -60,10 +60,8 @@ type IASCredentials struct {
 
 type UnifiedIdentityCredentials struct {
 	IASCredentials
-	AuthzURL         string                  `json:"authorization_url" validate:"required"`
-	AuthzBundleURL   string                  `json:"authorization_bundle_url" validate:"required_without=AuthzObjectStore"`
-	AuthzObjectStore *ObjectStoreCredentials `json:"authorization_object_store" validate:"required_without=AuthzBundleURL"`
-	AuthzInstanceID  string                  `json:"authorization_instance_id" validate:"required"`
+	AuthzURL        string `json:"authorization_url" validate:"required"`
```

Comment:
Can't we omit this completely and instead use the tenant url of the identity broker? Then we will use the tenant url instead of the default tenant, but that should make no difference.

Reply:
This change makes it, so we are using the tenant url. 
AuthzURL was unused, I removed it.